src/node_api.h

```diff
@@ -7,6 +7,10 @@
 
 struct uv_loop_s;  // Forward declaration.
 
+#ifndef NAPI_VERSION
```

Comment:
Add logic that makes the version number really high when experimental is requested.